linux_os/guide/system/network/network-kernel/network_host_and_router_parameters/sysctl_net_ipv4_conf_all_rp_filter/rule.yml

```diff
@@ -59,4 +59,10 @@ template:
     name: sysctl
     vars:
         sysctlvar: net.ipv4.conf.all.rp_filter
+        {{% if 'ol' in product or 'rhel' in product %}}
+        sysctlval:
+        - '1'
+        - '2'
```

Comment:
I think that the description and OCIL should be updated to describe that both 1 and 2 values are acceptable.

Reply:
Good catch!

Reply:
Updated based on your recent PR.